src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java

```diff
@@ -1893,10 +1893,12 @@ public void setSilentMode(boolean silentMode) {
 
     /**
      * Enable or disable Topic Association option.
+     * Replaced by {@link #setTopicAssociation(TopicAssociation)} ()
      *
      * @param enable true or false.
      */
     @Deprecated
+    @DataBoundSetter
```

Comment:
I think that's the way how we want it. If the old build configuration xml is read it will set already create the new TopicAssociation class.

Ich think line 2181 becomes then obsolete?

Reply:
`config.xml` deserialization doesn't care about `@DataBoundSetter` et.al. Those are only relevant for form submission and CasC.